### PR TITLE
Setup Seeder Files and Fix Migrations Issues

### PR DESCRIPTION
**Pull request by TundeMercy**
#### What does this PR do?
fix some issues on models, migrations and create seeders
#### How should this manually be Tested?
- `git clone` __https://github.com/devcareer/Dae-Alright-Backend__
- `cd` **Dae-Alright-Backend**
- `git checkout ch-update-db-setup`
- `npm install`
- `npm run db:migrate`
- `npm run db:seed`

### CODE REVIEW

File: src/database/models/vendor.js [hunk 1]
```diff
@@ -13,27 +16,59 @@ export default (sequelize, DataTypes) => {
...
       field: 'updated_at',
     },
   }, {});
 
+  Vendor.beforeCreate(async vendor => {
+    vendor.password = await vendor.generatePasswordHash();
+  });
+
+  Vendor.prototype.generatePasswordHash = async function generatePasswordHash() {
+    const saltRounds = +process.env.SALT;
```
Comment (Endowmissy) [inline src/database/models/vendor.js:35]: Why is there + in front of process.env.SALT

**Review by Endowmissy — APPROVED** [review]

File: src/database/models/vendor.js (hunk 1 above)
Comment (meetKazuki) [reply src/database/models/vendor.js:35]: Our `process.env.SALT` value is a number. We are putting the `+` sign to cast it to an integer

**Review by meetKazuki — APPROVED** [review]
LGTM.

**Review by maestro-1 — COMMENTED** [review]
Everything appears to be working okay

**Pull request merged**